string sensor_name
uint32 incubator_id
